File: app/src/main/java/com/aykuttasil/sweetloc/fragment/NavFragment.java
package com.aykuttasil.sweetloc.fragment;

import android.view.WindowManager;

import com.aykuttasil.sweetloc.R;
import com.aykuttasil.sweetloc.activity.MainActivity;

import org.androidannotations.annotations.AfterViews;
import org.androidannotations.annotations.EFragment;

/**
 * Created by aykutasil on 5.12.2016.
 */
@EFragment(R.layout.fragment_nav_layout)
public class NavFragment extends BaseFragment {

    @AfterViews
    @Override
    void initializeAfterViews() {

    }
}

File: app/src/main/java/com/aykuttasil/sweetloc/ui/fragment/NavFragment.kt
package com.aykuttasil.sweetloc.ui.fragment

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import com.aykuttasil.sweetloc.R

/**
 * Created by aykutasil on 5.12.2016.
 */
open class NavFragment : BaseFragment() {

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.fragment_nav_layout, container, false)
    }

}
